docs/usage/config-presets.md

```diff
@@ -37,21 +37,31 @@ In order to achieve these goals, preset configs allow for a very modular approac
 
 ## Preset Hosting
 
-In general, GitHub, GitLab or Gitea-based preset hosting is easier than npm because you avoid the "publish" step - simply commit preset code to the default branch and it will be picked up by Renovate the next time it runs.
+In general, GitHub, GitLab or Gitea-based preset hosting is easier than npm because you avoid the "publish" step - simply commit preset code to the default branch and it will be picked up by Renovate the next time it runs. You can also set a git tag (like a semver) to use specific releases of your shared config.
```

Comment:
```suggestion
In general, GitHub, GitLab or Gitea-based preset hosting is easier than npm because you avoid the "publish" step - simply commit preset code to the default branch and it will be picked up by Renovate the next time it runs.
You can also set a Git tag (like a SemVer) to use specific releases of your shared config.
```

Use one sentence per line. 😉 
Also capitalize proper nouns like Git, and also appropriately capitalize the term SemVer.